src/BloomTests/Book/TranslationGroupManagerTests.cs

```diff
@@ -228,5 +228,40 @@ public void PrepareElementsOnPage_HasLabelElementInsideTranslationGroup_LeavesUn
 			AssertThatXmlIn.Dom(dom).HasSpecifiedNumberOfMatchesForXpath("//label[@class='bloom-bubble']", 1);
 			AssertThatXmlIn.Dom(dom).HasSpecifiedNumberOfMatchesForXpath("//label[@class='bloom-bubble' and text()='something helpful']", 1);
 		}
+
+		[Test]
+		public void UpdateContentLanguageClasses_PrototypeElementHasImageContainer_ImageContainerCopiedToNewSybling()
+		{
+			const string contents = @"<div class='bloom-page'>
+										<div class='bloom-translationGroup'>
+											<div class='bloom-editable' lang='123'>
+												Do not copy me.
+												<br>Do not copy me.</br>
+												<p>Do not copy me.</p>
+												<div class='bloom-imageContainer'>
+													<img src='foo.png'></img>
+													<div contentEditable='true' class='caption'>Do not copy me</div>
+												</div>
+												Do not copy me.
+												<p>Do not copy me.</p>
+												<p class='foo bloom-cloneToOtherLanguages bar'>Do copy me.</p>
+											</div>
+										</div>
+									</div>";
+			var dom = new XmlDocument();
+			dom.LoadXml(contents);
+
+			TranslationGroupManager.PrepareElementsInPageOrDocument((XmlElement)dom.SafeSelectNodes("//div[contains(@class,'bloom-page')]")[0], _collectionSettings.Object);
+
+			AssertThatXmlIn.Dom(dom).HasSpecifiedNumberOfMatchesForXpath("//div[@lang='fr']/div/img[@src='foo.png']", 1);
```

Comment:
This made absolutely no sense to me until I finally figured out (after about 20 min) that (I think) //div[@lang='fr'] selects the copy that is supposed to be added of the bloom-editable div, which should have all the structure including a copy of the image container div, but none of the text except from the bloom-cloneToOtherLanguages paragraph. A comment to that effect would help.